pkg/apis/submarinerconfig/v1alpha1/types.go

```diff
@@ -73,12 +83,6 @@ type SubmarinerConfigSpec struct {
 	// GatewayConfig represents the gateways configuration of the Submariner.
 	// +optional
 	GatewayConfig `json:"gatewayConfig,omitempty"`
-
-	// LoadBalancerEnable enables or disables load balancer mode. When enabled, a LoadBalancer is created in the
-	// submariner-operator namespace (default false).
-	// +optional
-	// +kubebuilder:default=false
-	LoadBalancerEnable bool `json:"loadBalancerEnable"`
```

Comment:
Move this close to `NATTEnable` to fix the `golangci-lint` field alignment error.